pkg/action/archive.go

```diff
@@ -140,6 +140,26 @@ func extractTar(ctx context.Context, d string, f string) error {
 		return nil
 	case strings.Contains(filename, ".bz2") || strings.Contains(filename, ".bzip2"):
```

Comment:
While you are here, `.tbz` is a less common short extension form of `.tar.bz2`. If you add support for it, it might inspire a slightly different refactoring of this code.

Reply:
Added in [`c32840a` (#702)](https://github.com/chainguard-dev/malcontent/pull/702/commits/c32840af4cba55983f37e6e233c37170267ba06b).